packages/terra-form-checkbox/src/Checkbox.jsx

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './Checkbox.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+    * The checked property of the Input element. Use this to generate a controlled Checkbox Element.
+    */
+  checked: PropTypes.bool,
+  /**
+    * The checked property of the Input element. Use this to generate an uncontrolled Checkbox Element.
+    */
+  defaultChecked: PropTypes.bool,
+  /**
+    * The id of the input field.
+    */
+  id: PropTypes.string,
+  /**
+    * Additional attributes for the input object.
+    */
+  inputAttrs: PropTypes.object,
+  /**
+   * Whether the checkbox element is disabled or not.
+   */
+  isDisabled: PropTypes.bool,
+  /**
+    * Whether the checkbox element is inline or not.
+    */
+  isInline: PropTypes.bool,
+  /**
+    * Whether the label is hidden or not.
+    */
+  isLabelHidden: PropTypes.bool,
+  /**
+    * Text of the label.
+    */
+  labelText: PropTypes.node.isRequired,
```

Comment:
Any particular reason this is `labelText` instead of [`label`](https://github.com/cerner/terra-core/blob/master/packages/terra-form-field/src/Field.jsx#L50) or [`text`](https://github.com/cerner/terra-core/blob/master/packages/terra-button/src/Button.jsx#L49)?

Reply:
I believe this just came from what we had before for the [control component](https://github.com/cerner/terra-core/blob/master/packages/terra-form/src/Control.jsx#L40)